src/converters/accessor.ts

```diff
@@ -0,0 +1,45 @@
+import { YamlModel } from '../interfaces/YamlModel';
+import { Node } from '../interfaces/TypeDocModel';
+import { AbstractConverter } from './base';
+import { typeToString } from '../idResolver';
+import { Context } from './context';
+
+export class AccessorConverter extends AbstractConverter {
+    public convert(node: Node, context: Context): Array<YamlModel> {
+        const uid = context.ParentUid + '.' + node.name;
+        console.log(` - ${node.kindString}: ${uid}`);
+        let signatureType;
+        if (node.getSignature) {
+            if (Array.isArray(node.getSignature)) {
+                signatureType = node.getSignature[0].type;
+            } else {
+                signatureType = node.getSignature.type;
+            }
+        } else if (node.setSignature) {
+            if (Array.isArray(node.setSignature)) {
+                signatureType = node.setSignature[0].type;
+            } else {
+                signatureType = node.setSignature.type;
+            }
+        }
+
+        const model: YamlModel = {
+            uid: uid,
+            name: node.name,
+            fullName: node.name,
+            children: [],
+            langs: ['typeScript'],
```

Comment:
What is the langs usage at here?

Reply:
it's for the 'langs' field, which looks likes a fixed value. Here is an example: https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/master/docs-ref-autogen/%40azure/arm-appservice/Address.yml#L13-L14

Reply:
For this PR, I'd like to make minimal changes for the refactor phase 1, and keep the code logic unchanged if it won't impact the refactor. 

In next phase, I could move these fixed values to a better place instead of setting it everywhere...